src/store/middlewares/loggingMiddleware.js

```diff
@@ -0,0 +1,13 @@
+const logger = ({ getState }) => {
```

Comment:
I don't get the purpose of this one. Redux DevTools are showing exactly this information :)

Reply:
Well I am still obsessed with breakpoints and this is a place where I can put my breakpoint. Also for debugging other browsers than chrome. And I also wanted to try to work with some middlewares.